backends/gdx-backend-lwjgl3/src/com/badlogic/gdx/backends/lwjgl3/Lwjgl3Graphics.java

```diff
@@ -534,6 +534,36 @@ public void setSystemCursor (SystemCursor systemCursor) {
 		Lwjgl3Cursor.setSystemCursor(getWindow().getWindowHandle(), systemCursor);
 	}
 
+	public void setIdleFPS(int idleFPS) {
+		getWindow().getConfig().idleFPS = idleFPS;
+	}
+
+	public void setMSAASamples(int samples) {
+		Lwjgl3ApplicationConfiguration config = getWindow().getConfig();
+		setBackBufferConfig(config.r, config.g, config.b, config.a, config.depth, config.stencil, samples);
+	}
+
+	public void setColorBits(int r, int g, int b, int a) {
+		Lwjgl3ApplicationConfiguration config = getWindow().getConfig();
+		setBackBufferConfig(r, g, b, a, config.depth, config.stencil, config.samples);
+	}
+
+	public void setDepthBits(int depth) {
+		Lwjgl3ApplicationConfiguration config = getWindow().getConfig();
+		setBackBufferConfig(config.r, config.g, config.b, config.a, depth, config.stencil, config.samples);
+	}
+
+	public void setStencilBits(int stencil) {
+		Lwjgl3ApplicationConfiguration config = getWindow().getConfig();
+		setBackBufferConfig(config.r, config.g, config.b, config.a, config.depth, stencil, config.samples);
+	}
```

Comment:
I have to say, I don't like the way this is structured. Each call to set some part of the back buffer config recreates the window, potentially happening more than once if something makes multiple calls to set different parts. The main way the configuration is currently used is via `setBackBufferConfig()`, so I don't know why we really need new methods that only partially set that config. I know these respect the current settings for config that isn't being changed, but if the docs are clear on the defaults, and developers save the things they want saved, I think it might avoid any potentially strange behavior from the graphics getting repeatedly created if the only way to change the back buffer config is by `setBackBufferConfig()`. If there aren't any possible issues from repeatedly creating the graphics, just ignore this.

Reply:
>The main way the configuration is currently used is via setBackBufferConfig(), so I don't know why we really need new methods that only partially set that config.

I did it like it is now because it felt more natural to me & more in line with the rest of `Lwjgl3Graphics` but of course that can be changed - it's just how I structure code.

>If there aren't any possible issues from repeatedly creating the graphics, just ignore this.

I haven't noticed any strange behavior from repeatedly creating the window, other than the game slowing down to a crawl tbh

Reply:
>I haven't noticed any strange behavior from repeatedly creating the window, other than the game slowing down to a crawl. It does look pretty bad though - I'll add something to Lwjgl3RecreateWindowTest later to test for this case.

I brainfarted, I take it back. It absolutely creates a problem. I'm on it.